apps/system/js/app_chrome.js

```diff
@@ -698,9 +651,6 @@
         return;
       }
 
-      // We havent got a name for this location
-      this._gotName = false;
```

Comment:
https://github.com/benfrancis/gaia/blob/1168961/apps/system/js/app_chrome.js#L454

I think you need to remove this line as well